tutorial/westeros/westeros.ipynb

```diff
@@ -0,0 +1,1268 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "beta = 0.7 # an elasticity of demand\n",
+    "gdp = pd.Series([1., 1.21631, 1.4108], index=pd.Index(horizon, name='Time'))\n",
```

Comment:
do you know how these values were generated? might be good to be explicit here...

Reply:
yes, will replace with global gdp trajectory for clarity

Reply:
done!